clang/runtime/dpct-rt/include/util.hpp.inc

```diff
@@ -51,6 +51,26 @@
 
 namespace dpct {
 
+// DPCT_LABEL_BEGIN|err_tmp_types|dpct
+// DPCT_DEPENDENCY_EMPTY
+// DPCT_CODE
+namespace detail {
+
+template <typename tag, typename T> class new_type {
```

Comment:
`new_type` is confusing. We need a better name such as `generic_error_type` or `error_type_base`.